lib/src/features/ConnectionNegotiationManager.dart

```diff
@@ -107,21 +107,24 @@ class ConnectionNegotiationManager {
     supportedNegotiatorList
         .add(SaslAuthenticationFeature(_connection, _accountSettings.password));
     if (streamManagement.isResumeAvailable()) {
+      supportedNegotiatorList
```

Comment:
@slashdigital this one having error because stream management init two times from the old code for resume